### DISCUSSION TITLE
Link Preview

### DISCUSSION THREAD

**Discussion by markox92** [zen-browser/desktop#1055] [Ideas]
This is an extension in Firefox, but I want to see native support in Zen. It's can be an amazing feature. Something like this have Arc browser also.

https://github.com/user-attachments/assets/2fdb6c2e-7faf-4e11-a76f-4cb5ce972ee4

**Comment by sidsharths1245** [comment]
Which extension is this?

**Reply by badjuice** [reply]
I am currently using one called `MaxFocus: Link Preview`.

It's available on both Firefox and Chrome stores.

**Comment by Anthonyy232** [comment]
This would be fantastic, similar peeking feature to Arc

**Comment by universish** [comment]
It's not a very necessary feature. If you really want it, 
right click > open in new window.
